### PR TITLE
Fix some issues, found by Coverity Scan

### PR DESCRIPTION
**Pull request by akortunov**
Monstly missing valiables initialization and missing checks for null.
Probably we will need to rescan after merge.

### CODE REVIEW

File: apps/openmw/mwgui/container.cpp [hunk 1]
```diff
@@ -82,7 +82,7 @@ namespace MWGui
             dialog->eventOkClicked.clear();
             dialog->eventOkClicked += MyGUI::newDelegate(this, &ContainerWindow::dragItem);
         }
-        else
+        else if (mModel)
```
Comment (zinnschlag) [inline apps/openmw/mwgui/container.cpp]: That is an odd place to fix it. Had to look up the source to understand what is going on. IMHO the check mModel would be better placed inside of dragItem.

File: components/esm/loadcell.hpp [hunk 2]
```diff
@@ -78,18 +78,29 @@ struct Cell
...
+      AMBIstruct() : mAmbient(0),
+                     mSunlight(0),
+                     mFog(0),
+                     mFogDensity(0.f)
+      {}
   };
 
   Cell() : mName(""),
            mRegion(""),
+           mData(DATAstruct()),
```
Comment (zinnschlag) [inline components/esm/loadcell.hpp]: These lines looks pointless. A struct will be initialised via the default constructor anyway, even if you don't list it in the initialiser list.

File: apps/openmw/mwgui/container.cpp (hunk 1 above)
Comment (akortunov) [reply apps/openmw/mwgui/container.cpp]: Moved.
Now I check if model exists inside dragItem() and dragItem() calls.

File: components/esm/loadcell.hpp (hunk 2 above)
Comment (akortunov) [reply components/esm/loadcell.hpp]: I removed this commit.

**Pull request merged**